<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerstab</class>
 <widget class="QWidget" name="customerstab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pb_new_pr">
     <property name="text">
      <string>Новый</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="leSearchPr">
     <property name="placeholderText">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbReport">
     <property name="text">
      <string>Отчет</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pbSearch">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_first_name"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_company_name"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Телефон</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="pb_order">
        <property name="text">
         <string>Покупки</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QPushButton" name="pbEditPr">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="le_mail"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Почта</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="le_index"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_last_name"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="le_phone"/>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pbDeletePr">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="le_city"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Индекс</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название компании</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Город</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pbUpdateColumn">
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="4">
    <widget class="QTableWidget" name="twCustomers"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
